5-hive

```diff
@@ -0,0 +1,163 @@
+Analysing New York taxis with RHadoop
+========================================================
+  author: Andrie de Vries & Michele Usuelli
+date: 2015-07-01, UseR!2015
+width: 1680
+height: 1050
+css: css/custom.css
+
+
+
+Introduction to Hive
+====
+  
+- Hadoop data warehouse
+- SQL-like interface
+- Language: HQL (Hive Query Language)
+- HQL translated into MapReduce jobs
+- R interface: RHive, RevoScaleR
+- Hue web interface
+
+
+
+
+Computing the number of taxi rides by hour
+====
+  
+Using Hive, we can replicate the rmr2 example counting the number of taxi runs by hour. The steps are
+
+1. Importing the data
+2. Querying the data
+
+
+
+
+Defining an external table
+====
+
+Starting from our data, we can create a Hive table. Since the data is already in a CSV format, the easiest option is creating an external table.
+
+We need to specify
+
+- The data location
+- The data format
+- The field formats
+
+
+
+
+Query to create an extenal table
+====
+
+```
+CREATE EXTERNAL TABLE taxi_sample(medallion STRING, hack_license STRING, vendor_id STRING, rate_code INT, store_and_fwd_flag STRING, pickup_datetime STRING, dropoff_datetime STRING, passenger_count INT, trip_time_in_secs INT, trip_distance FLOAT, pickup_longitude FLOAT, pickup_latitude FLOAT, dropoff_longitude FLOAT, dropoff_latitude FLOAT) ROW FORMAT
```

Comment:
As far as I can tell you are demonstrating hive here, not RHive.  Is there a reason you don't include the R code?